tinybytes/src/lib.rs

```diff
@@ -291,6 +320,128 @@ impl Serialize for Bytes {
     }
 }
 
+pub struct RefCountedCell {
+    raw: RawRefCountedCell,
```

Comment:
What's the point of the nested struct? Can't you just put data and vtable right in here?